examples/vault-ec2-auth/README.md

```diff
@@ -0,0 +1,171 @@
+# Vault authentication using EC2 metadata example
+
+This example shows how to use the metadata from an EC2 instance to authenticate
+to a [vault cluster][vault_cluster].
```

Comment:
This repo lives in the Terraform Registry, which has an unfortunate requirement: all links must be absolute.